src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -29,10 +29,17 @@ function exit_handler()
     "[DEBUG]" "Docker container exit handler: EXIT signal received in docker container, exiting ..."
 }
 
+function kill_handler()
+{
+  printf "%s %s\n" \
+    "[INFO]" "Docker container killed due to disk pressure. If your job needs large disk space, please use HDFS or NFS to store your data."
```

Comment:
Container killed probably due to disk or memory pressure.

Reply:
After experiment, I changed the signal to SIGUSR1(10) and the container can handle the signal as expected. So if we trapped signal 10, means the container is killed due to disk pressure.